automatic/octave.portable/tools/chocolateyInstall.ps1

```diff
@@ -5,24 +5,20 @@ $version = '8.2.0'
 $toolsDir = $(Split-Path -parent $MyInvocation.MyCommand.Definition)
 $progDir  = "$toolsDir\octave"
 
-$osBitness = Get-OSArchitectureWidth
 
 $packageArgs = @{
   PackageName    = 'octave.portable'
   UnzipLocation  = $toolsDir
-  Url            = 'https://ftp.gnu.org/gnu/octave/windows/octave-8.2.0-w32.7z'
   Url64          = 'https://ftp.gnu.org/gnu/octave/windows/octave-8.2.0-w64.7z'
-  Checksum       = '9d6a81d86d7128775f8e821d89704582da81cee51400071c6d099118c983f37a'
   Checksum64     = '6c07a7e5cf748e2efc2ae719dd8ad9b07e41cf7abb645b84cf753b6e0cc2bfd4'
-  ChecksumType   = 'sha256'
   ChecksumType64 = 'sha256'
 }
 
 Install-ChocolateyZipPackage @packageArgs
 
 # Rename unzipped folder
-If (Test-Path "$toolsDir\octave-$version-w$osBitness") {
-  Rename-Item -Path "$toolsDir\octave-$version-w$osBitness" -NewName 'octave'
+If (Test-Path "$toolsDir\octave-$version-w64") {
+  Rename-Item -Path "$toolsDir\octave-$version-w64" -NewName 'octave'
 }
 If (Test-Path "$toolsDir\octave-$version") {
```

Comment:
I'm confused as to why this has been left in? If the binaries are all `octave-$version-w64`, this is not needed and does cause confusion (for me)?